### PR TITLE
Initial commit of aws-actionmailbox-ses

### PR DESCRIPTION
**Pull request by mullermp**
Includes ActionMailbox from aws-sdk-rails as its own gem version 0.1.0.

Migration plan: aws-sdk-rails ~> 4 will depend on this with backwards compatibility, but aws-sdk-rails ~> 5 will not.

### CODE REVIEW

**Comment by mullermp** [comment]
@ssunday

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+env:
+  ruby_version: 3.3
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7, '3.0', 3.1, 3.2, 3.3, jruby-9.4]
```
Comment (ssunday) [inline .github/workflows/ci.yml:21]: drop ruby 2.7?

Comment (ssunday) [reply .github/workflows/ci.yml:21]: and 3.0

Comment (ssunday) [reply .github/workflows/ci.yml:21]: I mean, since we're clean slating it.

**Review by ssunday — COMMENTED** [review]
I'll need to get time to try this out on a project. It seems like we couldn't use an old version of aws-sdk-rails with this, as it's conflict/overwrite each other? We would need to test with the new version of it with this as a requirement, right?

**Comment by ssunday** [comment]
Why is task/release a submodule? Or what is that?

Looking generally good.

**Comment by mullermp** [comment]
It's shared release tooling that we've made to publish gems and documentations for each github repo. It's in aws/aws-sdk-ruby-release-tools if you are curious.

**Review by alextwoods — APPROVED** [review]

**Pull request merged**